src/routes/auth.js

```diff
@@ -431,13 +432,11 @@ router.post('/requestToken', async (req, res) => {
 
 router.get('/verifyRToken', async (req, res) => {
     try {
-        console.log('here');
         const { q } = req.query;
         const { requestToken } = decode(q);
```

Comment:
Is `q` a JWT token? If yes then you need to verify it. If no, then just use `const {reqToken} = req.body`. I think there is no need for JWT signing at this stage since this parameter will be publicly visible in the URL. The `/getAuthToken` method would then check if the request token correctly maps to the client that is trying to exchange it. This would ensure that even if the request token is compromised, a malicious entity would not be able to get access to the auth token.